providers/okta/okta.go

```diff
@@ -92,6 +94,60 @@ func (c *Client) GroupByName(ctx context.Context, groupName string) (*okta.Group
 	return nil, fmt.Errorf("unable to find okta group %q", groupName)
 }
 
+func (c *Client) GroupsByName(ctx context.Context, groupNames []string, batchsize int) ([]*okta.Group, error) {
```

Comment:
Note: There is a mix of [pointer](https://github.com/kanopy-platform/go-library/blob/bda8c60e03b1c80a7d6210f52987fe3f8a40f680/providers/okta/okta.go#L79) and [non-pointer](https://github.com/kanopy-platform/go-library/blob/bda8c60e03b1c80a7d6210f52987fe3f8a40f680/providers/okta/okta.go#L56) return types in this package. This should be consistent, and non-pointer seems more aligned with the underlying SDK.

Reply:
I'll take a look at this.